app/src/main/java/org/wikipedia/main/MainFragment.kt

```diff
@@ -486,39 +487,36 @@ class MainFragment : Fragment(), BackPressedHandler, FeedFragment.Callback, Hist
     private fun maybeShowEditsTooltip() {
         if (currentFragment !is SuggestedEditsTasksFragment && Prefs.shouldShowSuggestedEditsTooltip() &&
                 Prefs.getExploreFeedVisitCount() == SHOW_EDITS_SNACKBAR_COUNT) {
```

Comment:
Since the general tooltip for Edits tab will show only when the count equals to `SHOW_EDITS_SNACKBAR_COUNT`, should we add an extra logic to the `maybeShowImageRecsTooltip` with `Prefs.getExploreFeedVisitCount() > SHOW_EDITS_SNACKBAR_COUNT`?

Reply:
Agreed